<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoringMeasuresForm</class>
 <widget class="QWidget" name="ScoringMeasuresForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>815</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fibers</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="fibersCombo">
       <item>
        <property name="text">
         <string>None selected</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="outputLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="dataDependentGroupBox">
     <property name="title">
      <string>Data dependent (external energy)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Glyph data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="glyphDataCombo">
          <item>
           <property name="text">
            <string>None selected</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dataIndependentGroupBox">
     <property name="title">
      <string>Data independent (internal energy)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lambdaTopLabel">
        <property name="text">
         <string>Weight of independent term</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lambdaLabel">
          <property name="text">
           <string> λ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lambdaSlider">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="lambdaSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Length punishment</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>β</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="betaSlider">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="betaSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Torsion punishment</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>μ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="muuSlider">
          <property name="minimum">
           <number>-500</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="muuSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Type of curve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="typeCurveCombo">
          <item>
           <property name="text">
            <string>Geodesic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Elastica</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="applyLogarithmCheckBox">
        <property name="text">
         <string>Apply logarithm</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="normalizeGlyphDataCheckBox">
        <property name="text">
         <string>Normalize glyph data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="standardizeScalarsCheckBox">
        <property name="text">
         <string>Standardize output</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>251</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
